### DISCUSSION TITLE
Seems like Personal Access Token is not working suddenly when I do git push and pull

### DISCUSSION THREAD

**Discussion by zawhtutwin** [community/community#57213] [New to GitHub] [answered]
Seems like Personal Access Token is not working suddenly when I do git push and pull. Even before today, I could do all git operation with personal access token but today cannot. Please help.

**Comment by zawhtutwin** [comment, marked as answer]
I restarted my Ubuntu and it's working fine now

**Reply by queenofcorgis** [reply]
Happy to hear so! Thanks for coming back to the post and letting us know the solution.

**Comment by hopa-tkd** [comment]
Seems like Personal Access Token is not working suddenly when I do git push and pull
I changed my token and want to re-authenticate
brings me this
remote: Support for password authentication was removed on August 13, 2021.
and this is
remote: Please see https://docs.github.com
and this is
fatal: Authentication failed for 'https://github.com

And when I try to add a token in place of the password
remote: Repository not found.


please heeeeeeeeeeeeeeelp

**Reply by B-Phan** [reply]
Did you manage to find a fix?

**Comment by mervynstunner** [comment]
These tokens are total headache

**Comment by kenza1111** [comment]
Try to use this command: jupyter notebook --no-browser --port 9888.
It is now working for me

**Comment by minfrin** [comment]
Not working for me either, was working fine yesterday?

```
[minfrin@localhost xx]$ git push
Username for 'https://github.com': minfrin
Password for 'https://minfrin@github.com': 
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/xx/xx/'
```

**Reply by minfrin** [reply]
Wait one day, and the same PAT works fine. Looks like instability on the Github side.

**Comment by Spidey03** [comment]
The Personal Access Token (PAT) may have expired. Please try generating a new access token.

**Comment by dorthyuser** [comment]
Hi,
I am facing same issue, once this works and next time when you want to show something using this, this is gone.
Is Github doing something for this.

Note: My token has not expired when running the command with several days of life still left.

Thanks.
Do

**Comment by shibbal66** [comment]
I've created multiple access tokens for github but none of them seem to work. It was working fine till yesterday.

**Comment by brannanjp** [comment]
I'm experiencing this problem today (July 3, 2026) The error messages I'm getting is:
remote: Invalid username or token. Password authentication is not supported for Git operations.
